README.md

```diff
@@ -67,51 +68,52 @@ This table provides information about the software prerequisites for the Ansible
 
 | **Ansible Modules** | **OneFS Version** | **Python version** | **Python SDK version** | **Ansible**              |
 |---------------------|-----------------------|--------------------|----------------------------|--------------------------|
-| v2.4.1 | 9.3.x <br> 9.4.x <br> 9.5.x | 3.9 <br> 3.10 <br> 3.11 | 9.3.0 <br> 9.4.0 <br> 9.5.0 | 2.14 <br> 2.15 <br> 2.16 |
+| v2.5.0 | 9.3.x <br> 9.4.x <br> 9.5.x | 3.9 <br> 3.10 <br> 3.11 | 9.3.0 <br> 9.4.0 <br> 9.5.0 | 2.14 <br> 2.15 <br> 2.16 |
```

Comment:
change SDK version to 0.3.0.1 instead of 9.3.0 <br> 9.4.0 <br> 9.5.0

Reply:
updated